department.rb

```diff
@@ -0,0 +1,42 @@
+require './employee'
+
+class Department
+  attr_reader :dept_name
+
+  def initialize(dept_name)
+    @dept_name = dept_name
+    @employees = []
+  end
+
+  def add_employee(employee)
+    @employees << employee
+  end
+
+  def employees
+    @employees
+  end
+
+  def salary_list
+    sum = 0
+    @employees.each { |employee| sum += employee.salary }
```

Comment:
Good logic.  Recall that you could also do lines 20 and 21 together in a `reduce` call.